client/src/components/Login.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { signApi } from "../api/auth";
+
+const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState(null);
+  const handleChange = event => {
+    if (event.target.name === "email") {
+      setEmail(event.target.value);
+    }
+    if (event.target.name === "password") {
+      setPassword(event.target.value);
+    }
+  };
+  const signIn = () => {
+    signApi(email, password)
+      .then(data => {
+        const token = data.token;
+        console.log(token);
+
+        localStorage.setItem("token", token);
+        localStorage.setItem("user", JSON.stringify(data.user));
+        document.location.reload();
+      })
+      .catch(() => {
+        setError(true);
+      });
+  };
+  const token = localStorage.getItem("token");
+  if (token) {
+    return <div>You are already logged in.</div>;
+  } else
+    return (
+      <div>
```

Comment:
It's a good idea to wrap inputs and submit buttons in a `form`, and to provide a label for every input, so let's do that. Also, let's use Semantic UI components! 

Here's an example of creating a form with Semantic UI: https://react.semantic-ui.com/collections/form/#types-form

Reply:
Since this `div` isn't needed, let's remove it!